File: docker-selenium/issue-1416/issue_1416.py
# docker run -p 4444:4444 -p 7900:7900 --shm-size="2g" selenium/standalone-firefox:4.0.0-20211102
# pip install selenium -> 4.0.0
import time
from selenium import webdriver

firefox_options = webdriver.FirefoxOptions()
firefox_options.set_preference('intl.accept_languages', 'nl-NL, nl')
driver = webdriver.Remote(
    command_executor='http://localhost:4444',
    options=firefox_options
)
driver.get("https://www.google.com")
time.sleep(5)
driver.quit()

File: docker-selenium/issue-1416/src/test/java/Issue1416.java
import org.openqa.selenium.UnexpectedAlertBehaviour;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxOptions;
import org.openqa.selenium.remote.RemoteWebDriver;

import java.net.MalformedURLException;
import java.net.URL;

public class Issue1416 {

  public static void main(String[] args) throws MalformedURLException, InterruptedException {
    FirefoxOptions options = new FirefoxOptions();
    options.setUnhandledPromptBehaviour(UnexpectedAlertBehaviour.ACCEPT);
    options.addPreference("intl.accept_languages", "nl-NL, nl");

    WebDriver driver = new RemoteWebDriver(new URL("http://localhost:4444"), options);

    try {
      driver.get("https://www.google.com");

      // Check the browser, it should be in de-DE
      Thread.sleep(5000);
    } finally {
      driver.quit();
    }

  }

}
